Add Funding Format (#21)
interchainio/funding: README.md

We often work with applicants to revise their proposed scope to 
ensure success and better meet the needs of the ecosystem.

## Funding Formats

Funding can take shape in one of three formats:

>  * **Service Agreement** - This is the most common format and includes clear deliverables-based payments.

Reviewer: ```suggestion
 * **Service Agreement** - This type is the most common, with payments based on clear fulfillment of deliverables.
```

>  * **Grants** - This format is used when deliverables are less clear or work is part of a larger ongoing process with potentially many stake holders.

Reviewer: ```suggestion
 * **Grants** - This type of financing is preferred if deliverables are less discrete, if proposed work is part of an ongoing process or multi-stakeholder system where ICF oversight will have little effect on the outcome.
```

>  * **Investment** - This is the least common format and occurs when there is exceptional alignment between a project and the Interchain Foundation's Mission and Vision.

Reviewer: ```suggestion
 * **Investment** - This is the least common type and occurs when there is exceptional alignment between a project and the Interchain Foundation's Mission and Vision.
```
